scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -184,40 +231,152 @@ def parse_args(o3de_context: exp.O3DEScriptExportContext):
+                                           enable_override_arg=['-game', '--game-launcher'],
+                                           enable_override_desc="Enable the building and inclusion of the Game Launcher.",
+                                           disable_override_arg=['--nogame', '--no-game-launcher'],
+                                           disable_override_desc="Disable the building and inclusion of the Game Launcher if not needed.")
+
+        export_config.add_boolean_argument(parser=parser,
+                                           key=exp.SETTINGS_OPTION_BUILD_SERVER_LAUNCHER.key,
+                                           enable_override_arg=['-server', '--server-launcher'],
+                                           enable_override_desc="Enable the building and inclusion of the Server Launcher.",
+                                           disable_override_arg=['--noserver', '--no-server-launcher'],
+                                           disable_override_desc="Disable the building and inclusion of the Server Launcher if not needed.")
+
+        export_config.add_boolean_argument(parser=parser,
+                                           key=exp.SETTINGS_OPTION_BUILD_HEADLESS_SERVER_LAUNCHER.key,
+                                           enable_override_arg=['-headless', '--headless-server-launcher'],
+                                           enable_override_desc="Enable the building and inclusion of the Headless Server Launcher.",
+                                           disable_override_arg=['--noheadless', '--no-headless-server-launcher'],
+                                           disable_override_desc="Disable the building and inclusion of the Headless Server Launcher if not needed.")
+
+        export_config.add_boolean_argument(parser=parser,
+                                           key=exp.SETTINGS_OPTION_BUILD_UNIFIED_SERVER_LAUNCHER.key,
+                                           enable_override_arg=['-unified', '--unified-launcher'],
+                                           enable_override_desc="Enable the building and inclusion of the Unified Launcher.",
+                                           disable_override_arg=['--nounified', '--no-unified-launcher'],
+                                           disable_override_desc="Disable the building and inclusion of the Unified Launcher if not needed.")
+
+        export_config.add_boolean_argument(parser=parser,
+                                           key=exp.SETTINGS_OPTION_ENGINE_CENTRIC.key,
+                                           enable_override_arg=['-ec', '--engine-centric'],
+                                           enable_override_desc="Enable the engine-centric work flow to export the project.",
+                                           disable_override_arg=['-pc', '--project-centric'],
+                                           disable_override_desc="Enable the project-centric work flow to export the project.")
+
```

Comment:
Missing `platforms` argument
```suggestion
        parser.add_argument('-pl', '--platform', type=str, default=exp.get_default_asset_platform(), choices=['pc', 'linux', 'mac'])

```